--- a/Makefile.am
+++ b/Makefile.am
@@ -188,2 +188,3 @@
 	resize.c \
+	screen-compat.c \
 	screen-redraw.c \
--- a/tmux.h
+++ b/tmux.h
@@ -2413,6 +2413,9 @@
 const char	*find_home(void);
 const char	*getversion(void);
 
+/* screen-compat.c */
+void		 screen_compat_translate(int *, char ***);
+
 /* proc.c */
 struct imsg;
 int	proc_send(struct tmuxpeer *, enum msgtype, int, const void *, size_t);
--- a/tmux.c
+++ b/tmux.c
@@ -382,6 +382,8 @@
 	const struct options_table_entry	*oe;
 	u_int					 i;
 
+	screen_compat_translate(&argc, &argv);
+
 	if (setlocale(LC_CTYPE, "en_US.UTF-8") == NULL &&
 	    setlocale(LC_CTYPE, "C.UTF-8") == NULL) {
 		if (setlocale(LC_CTYPE, "") == NULL)
