--- a/tmux.c
+++ b/tmux.c
@@ -32,6 +32,7 @@
 #include <unistd.h>
 
 #include "tmux.h"
+#include "screen-to-tmux-translator"
 
 struct options	*global_options;	/* server options */
 struct options	*global_s_options;	/* session options */
@@ -382,6 +383,8 @@
 	const struct options_table_entry	*oe;
 	u_int					 i;
 
+	screen_to_tmux_translate(&argc, &argv);
+
 	if (setlocale(LC_CTYPE, "en_US.UTF-8") == NULL &&
 	    setlocale(LC_CTYPE, "C.UTF-8") == NULL) {
 		if (setlocale(LC_CTYPE, "") == NULL)
